gm4_metallurgy/data/arborenda_shamir/functions/transfer_shamir/add_band.mcfunction

```diff
@@ -0,0 +1,10 @@
+#@s = obsidian cast with a arborenda shamir on it
+#called from initiate_transfer
+
+#check whether the other item is valid
+scoreboard players set valid_tool gm4_ml_data 0
+execute as @e[type=item,dx=0,limit=1,nbt=!{Item:{tag:{gm4_metallurgy:{has_shamir:1b}}}}] run function arborenda_shamir:transfer_shamir/check_tool_validity
+
+execute if score valid_tool gm4_ml_data matches 1 run data merge entity @s {Item:{Count:1b,id:"minecraft:obsidian",tag:{display:{Lore:["Slightly damaged obsidian"]}}},Tags:["gm4_ml_smooshed"]}
+execute if score valid_tool gm4_ml_data matches 1 run data remove entity @s Item.tag.gm4_metallurgy
+execute if score valid_tool gm4_ml_data matches 1 run data remove entity @s Item.tag.SkullOwner
```

Comment:
This would create 13 additional functions, it's just not worth it for something as simple as 2 additional fake player score checks. It also just runs when someone is smooshing a band so the lag caused by this is marginal.